jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopup.java

```diff
@@ -164,9 +166,11 @@ public void setRequest(RequestSummary r,
             errorControlGroup.setVisible(true);
             String html = "";
             for (ErrorSummary error : errors) {
-                html += "<strong>" + error.getMessage() + "</strong><br/>" + error.getStacktrace() + "<br><br>";
+                html += error.getMessage() +
```

Comment:
Maybe it makes sense for easier readability reasons to also rename the variable from ```html`` to, for example, ```textArea``` so it's aligned with the field type used.